Cargo.toml

```diff
@@ -21,3 +21,4 @@ async-task = "4.0.2"
 atomic-waker = "1.0.0"
 fastrand = "1.3.4"
 futures-lite = "1.11.0"
+log = "0.4.17"
```

Comment:
this added non-optional dependency is probably not a good idea

Reply:
`log` already appears in `smol`'s dependency tree several times, so I don't think it's unjustified to add it here. In addition, I think that reporting the error emitted by the thread spawn failure is probably better than just ignoring it.

Reply:
I think `tracing` would maybe be a bit more appropriate, it is pretty widespread in the async ecosystem...

Reply:
I can replace it here if you want, but if we do this I think we should go ahead and update [`async-io`](https://github.com/smol-rs/async-io/blob/f9ba3f7f8a9263ad173cdd734e31e4003e57af59/Cargo.toml#L25) and [`polling`](https://github.com/smol-rs/polling/blob/bc56d1fb380aec713473049b130c28e2835c3a9f/Cargo.toml#L26) with `tracing` as well.

Reply:
ok, we can do that later (e.g. if we do breaking-change releases of such crates)

Reply:
I don't think this would be a breaking change, since the logging is entirely internal to the crates.

Reply:
hmm idk.